docs/modules/ROOT/pages/how-tos/cloudscale/decommission.adoc

```diff
@@ -0,0 +1,116 @@
+= Uninstallation on cloudscale.ch
+
+[abstract]
+--
+Steps to remove an OpenShift 4 cluster from https://cloudscale.ch[cloudscale.ch].
+--
+
+[NOTE]
+--
+- The commands are idempotent and can be retried if any of the steps fail.
+- In the future, this procedure will be mostly automated
+--
+
+== Prerequisites
+
+* `docker`
+* `mc` https://docs.min.io/docs/minio-client-quickstart-guide.html[Minio client] (aliased to `mc` if necessary)
+* `jq`
+* `yq` https://mikefarah.gitbook.io/yq[yq YAML processor]
+
+
+== Cluster Decommission
+
+. Export the following vars
++
+[source,console]
+----
+export CLOUDSCALE_TOKEN=<cloudscale-api-token> # From https://control.cloudscale.ch/user/api-tokens
+export CLUSTER_ID=<lieutenant-cluster-id>
+export TENANT_ID=<lieutenant-tenant-id>
+export REGION=<region> # rma or lpg (without the zone number)
+export GITLAB_TOKEN=<gitlab-api-token> # From https://git.vshn.net/profile/personal_access_tokens
+export GITLAB_CATALOG_PROJECT_ID=<project-id> # GitLab numerical project ID of the catalog repo
+----
+
+include::partial$setup_terraform.adoc[]
+
+. Delete resources from clouscale.ch using Terraform
++
+[source,console]
+----
+terraform destroy
+----
+
+. After all resources are deleted we need to remove the bucket
++
+[source,console]
+----
+# Use already exiting bucket user
+response=$(curl -sH "Authorization: Bearer ${CLOUDSCALE_TOKEN}" \
+  https://api.cloudscale.ch/v1/objects-users | \
+  jq -e ".[] | select(.display_name == \"${CLUSTER_ID}\")")
+
+# configure minio client to use the bucket
+mc config host add \
+  "${CLUSTER_ID}" "https://objects.${REGION}.cloudscale.ch" \
+  $(echo $response | jq -r '.keys[0].access_key') \
+  $(echo $response | jq -r '.keys[0].secret_key')
+
+# delete bootstrap-ignition object
+mc rb "${CLUSTER_ID}/${CLUSTER_ID}-bootstrap-ignition" --force
```

Comment:
Will this work if there's still content in the bucket?
I think you have to run `mc rm -r --force "${CLUSTER_ID}/${CLUSTER_ID}/"` first

Reply:
For me it worked